### ISSUE TITLE
Github commit konnectors and double authentication

### ISSUE THREAD

**Issue by EtienneM** [cozy-labs/konnectors#50]
I would like to use the Github commit konnector but I use the double authentication. Will the konnector work anyway? On my Cozy installation it says that the import has been done but I don't know how to see the imported data...

**Comment by frankrousseau** [comment]
You can see them in the Databrowser application (but the Databrowser is quite buggy).

You can see dataviz (graphs) in the KYou application. 

Sometimes I have weird results, I'm not sure it works properly.

**Comment by EtienneM** [comment]
I cannot find any imported commit in the data browser. So maybe the import was not successful because of the double authentication. However no message told me the status of the import operation.

**Comment by frankrousseau** [comment]
Yes, it's sure the konnector doesn't handle the double authentication. Sorry about that.

**Comment by frankrousseau** [comment]
I add the bug label because of the lack of message about the failure.
